dev-proxy/ConsoleLogger.cs

```diff
@@ -31,7 +32,7 @@ public ConsoleLogger(ProxyConfiguration configuration, PluginEvents pluginEvents
         LogLevel = configuration.LogLevel;
     }
 
-    public void LogInfo(string message)
+    private void LogInfo(string message)
```

Comment:
Since we're not exposing these methods outside of this logger now, it would be clearer to rename them to `WriteInfo`, etc. That way, we don't confuse with the methods that are a part of `ILogger` and are merely util methods for this particular sink.

Reply:
Yes